satpy/etc/enhancements/abi.yaml

```diff
@@ -29,3 +31,41 @@ enhancements:
           stretch: crude
           min_stretch: [-26.2, -43.2, 243.9]
           max_stretch: [0.6, 6.7, 208.5]
+
+  cira_fire_temperature:
+    standard_name: cira_fire_temperature
+    operations:
+      - name: stretch
+        method: !!python/name:satpy.enhancements.stretch
+        kwargs:
+          stretch: crude
+          min_stretch: [273.15, 0.0, 0.0]
+          max_stretch: [333.15, 100.0, 75.0]
+      - name: gamma
+        method: !!python/name:satpy.enhancements.gamma
+        kwargs:
+          gamma: [1.4, 1, 1]
+
+  cira_day_convection:
+    standard_name: cira_day_convection
+    operations:
+      - name: stretch
+        method: !!python/name:satpy.enhancements.stretch
+        kwargs:
+          stretch: crude
+          min_stretch: [-35.0, -5.0, -0.75]
+          max_stretch: [5.0, 60.0, 0.25]
```

Comment:
The B-channel stretch needs to be from `-75.0` to `25.0`, as Satpy handles reflectances as percentages.

Reply:
Ah. The values you inserted were as the documentation stated, but the documentation has them wrong (mix of fraction and %-units). The current values are very very close to each other, so will practically only yield either 0 or 255 for the blue channel pixel values. I've used locally `-70` ... `20`, which gives nice images, so the values certainly need to be in that scale.

Reply:
yes, i was doubting that stretch :), but obeyed the recipe. any thoughts on the AppVeyor fail on the PR?

Reply:
That seems to be a decoding error when loading the ABI sensor composites. I'd hazard a guess that changing `μm` to `um` in the `fire_temperature` description might help.

Reply:
Nevermind, the decoding error happens also on AHI compositor tests.

Reply:
stretch fixed.